### PR TITLE
Added measurement protocol guidance, and fixed image line colour

### PR DESCRIPTION
**Pull request by garanj**

### CODE REVIEW

File: docsource/docs/reference/measurement_protocol.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+The BigQuery table schema for `BQToMeasurementProtocol` is as follows:
+
+*   Field names and types should be names and types of [Measurement Protocol parameters](https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters).
+*   For Measurement Protocol parameters representing lists, [repeated fields](https://cloud.google.com/bigquery/docs/nested-repeated) (arrays) should be used (see details below).
+
+The only parameter that can be omitted is `v` — the Measurement Protocol version, which is provided by CRMint by default as `v=1`.
+
+If the table has a column with a name not matching a valid Measurement Protocol parameter, it will be sent as a non-existing parameter, resulting in an invalid Measurement Protocol hit. The idea behind this is to not interfere with what a user wants to send as Measurement Protocol hit: we're neither validating, nor transforming parameters and their names.
+
+The only exception is made for the names of repeated parameters: for those we *unfold* array fields to params named like `<field_name><array_element_idex>`.
```
[the remark this answers is not in the document]
Comment (garanj) [reply docsource/docs/reference/measurement_protocol.md]: Done

File: docsource/img/landing-2.svg [hunk 2]
```diff
@@ -1 +1 @@
...
...W+swXtbqd5XK87n7iXDSTIG7YEL35eCdqiy/VKqrnwC6zcncRxUIV74/+ck72tm0kgjwylH0ucW4G50ud9ATw2qkEdlEzuXruX9VtDzQVDCrbwEGSjVVY/vGTrvUd9Jb5F8ziaQVPTF39cHJVc6ivgGlD35YJywAcrJVl6H4W8q9YOBssuk9TM2V7X3Zp3rN4DOnrRqK/BgBIVIRl0/COJRL6AEu94OSt7IHo8G8CVN/KF6P7m16rW6mus2GogxG5LbRw8qKDa1ciwMEI9NyjXe4sFA3q1qR2X4P4uyUGJlU4L+tQb7+R9ETIhkrHaKhRFCmEUuO4+PqI0SsfY7qXZuC6GR/2wnsqKHCWB+Avll3wCfn4phlzCzhEvmnEuD+77m+8ki7K353UcgCZIQr3cJuQCpUMGbyCHIR8jzEIsasNM1kRMFFzvsQinUZlRq/0ji5BSFcRfmgLgx1IsxpL5MqqtzyIHB/w4sFDQVxlDYjBiK1qIDU1NZdeYf6cVgfznAwRbaMiUkIjwvNi7yWGHWhVkJlJOzSBRUgH8NJBexTgZO2QS+vx14HyGfLGgZRJH+uawoZ0RKabFzk5igoQksH4JceYyeLhNk254MZ8LfNgvfJhXyPJvAQj7NxOe/FgDR0Rbpu6CkSlDpPipURdmh/WWOfYa7604qS7LKxER2ivNZzX38qKY4ATpVrA0bJcERJ6XJLgR5/vYCg2sppO5rHwrxR+SwSDiqQyzMdPFoX6az0M+SXNPfZ0KkIJH7AhnQwJAKM0Wss0pkjnLxvnLCEncL58SdvA8ohRgoIIVKIBYK70JTnr1wC5PuNKWFBcbrvD7aJSFS6Npwo31zRnREOVCIYrGOm8fOOo1UIIAKGJzSrAP7go1H4DHCIco5ZLhofveUDcGr9Lk9QcnRSVUtF/KEOyadR58Rk6wVV6p+zgBrSWqU3DVPxHqy2BhJibKWZm1pBidFkMZnoBzlUYlHkuGEOD+OsxakOj3N2BHNcEOM1UBe0o8KWtXmUWK4KchBOkK8KHZFUniEGG4LUrv3TZHh7XhkGF6IsYwmr4dcN415ZBheCbKlSoxUhaIhjwrDS0EOEWKkWMrmPCIMrwU5Q8Q9duXRYPhBkFQFeBSPBMMvghzIrh2GnwTJYmQwGInhX1UIyZmvTR4EAAAAAElFTkSuQmCC"/></g></g></svg>
```
[the remark this answers is not in the document]
Comment (garanj) [reply docsource/img/landing-2.svg]: Done

File: docsource/docs/reference/measurement_protocol.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+For example, for multiple [Content Groups](https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters#cg_) you can either:
+
+*   Use separate fields named `cg1`, `cg2`, etc.
+*   **OR** use one array field named `cg` — this will be unfolded automatically to `cg1`, `cg2` and so on with values taken from the corresponding array elements (one-based indices are used).
+
+All of this is to support Enhanced E-Commerce [Product Impressions lists](https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters#il_nm) and [Product Lists](https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters#pr_id).
+
+Structures are  unfolded the same way as arrays are, but using structure keys instead of array indices.
+
```
[the remark this answers is not in the document]
Comment (garanj) [reply docsource/docs/reference/measurement_protocol.md]: Done

**Comment by garanj** [comment]
Changes made!

**Comment by Slony** [comment]
LGTM

**Pull request merged**